styles.css

```diff
@@ -0,0 +1,143 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@200;300;400;500;700&family=Roboto:wght@100;300;500;700&display=swap');
+
+* {
+  font-family: 'poppins', sans-serif;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  overflow-x: hidden;
+  height: 100%;
+  width: 100%;
+}
+
+@media screen and (max-width: 375px) {
```

Comment:
- [ ] Please remove this media query as it only applies to screen widths smaller than 375px. This width should be the min limit for you to consider styling. I recommend using only the min-width property when needed.

Reply:
hi   [tugsem](https://github.com/tugsem)] 
  ok i did some few changes please help me see the changes